platform-core-module/src/main/java/com/softicar/platform/core/module/maintenance/SessionOverviewPage.java

```diff
@@ -0,0 +1,46 @@
+package com.softicar.platform.core.module.maintenance;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.CoreModule;
+import com.softicar.platform.core.module.CoreRoles;
+import com.softicar.platform.core.module.module.instance.system.SystemModuleInstance;
+import com.softicar.platform.dom.node.IDomNode;
+import com.softicar.platform.emf.authorization.role.IEmfRole;
+import com.softicar.platform.emf.page.EmfPagePath;
+import com.softicar.platform.emf.page.IEmfPage;
+import com.softicar.platform.emf.source.code.reference.point.EmfSourceCodeReferencePointUuid;
+
+@EmfSourceCodeReferencePointUuid("2834bfc9-724b-4b30-9283-7c06660cf643")
+public class SessionOverviewPage implements IEmfPage<SystemModuleInstance> {
+
+	@Override
+	public Class<CoreModule> getModuleClass() {
+
+		return CoreModule.class;
+	}
+
+	@Override
+	public IDisplayString getTitle(SystemModuleInstance moduleInstance) {
+
+		return CoreI18n.SESSION_OVERVIEW;
+	}
+
+	@Override
+	public IEmfRole<SystemModuleInstance> getAuthorizedRole() {
+
+		return CoreRoles.MAINTENANCE_ADMINISTRATOR;
```

Comment:
why can you see the page, when nobody is assigned to it? we should assign the role to the admin user in the core test fixture, IMO

Reply:
well, I assigned myself the role under role assignment